src/Bicep.LangServer/Deploy/DeploymentParametersHelper.cs

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Bicep.LanguageServer.Configuration;
+using Bicep.LanguageServer.Handlers;
+using Newtonsoft.Json.Linq;
+
+namespace Bicep.LanguageServer.Deploy
+{
+    public class DeploymentParametersHelper
+    {
+        // Per documentation here- https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/parameter-files
+        // parameters file should be of below format:
+        //{
+        //  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+        //  "contentVersion": "1.0.0.0",
+        //  "parameters": {
+        //    "<first-parameter-name>": {
+        //      "value": "<first-value>"
+        //    },
+        //    "<second-parameter-name>": {
+        //      "value": "<second-value>"
+        //    }
+        //  }
+        //}
+        // However, azure-sdk-for-net expects parameters to be of name and value pairs:
+        // https://github.com/Azure/azure-sdk-for-net/blob/1e25b1bfc9b54df35d907aa7b2c10ff07082e845/sdk/resources/Azure.ResourceManager.Resources/src/Generated/Models/ArmDeploymentProperties.cs#L27
+        // We'll work around the above issue by first detecting the format of the file.
```

Comment:
Added this issue to remove support for just-property-bags: https://github.com/Azure/bicep/issues/7086